operators/dell-csm-operator/0.1.0/manifests/dell-csm-operator.v0.1.0.clusterserviceversion.yaml

```diff
@@ -0,0 +1,769 @@
+          - get
+          - list
+          - patch
+          - update
+          - watch
+        - apiGroups:
+          - storage.k8s.io
+          resources:
+          - volumeattachments/status
+          verbs:
+          - patch
+        serviceAccountName: dell-csm-operator-controller-manager
+      deployments:
+      - name: dell-csm-operator-controller-manager
+        spec:
+          replicas: 1
+          selector:
+            matchLabels:
+              control-plane: controller-manager
+          strategy: {}
+          template:
+            metadata:
+              labels:
+                control-plane: controller-manager
+            spec:
+              containers:
+              - args:
+                - --secure-listen-address=0.0.0.0:8443
+                - --upstream=http://127.0.0.1:8080/
+                - --logtostderr=true
+                - --v=10
+                image: gcr.io/kubebuilder/kube-rbac-proxy:v0.8.0
+                name: kube-rbac-proxy
+                ports:
+                - containerPort: 8443
+                  name: https
+                  protocol: TCP
+                resources: {}
+              - args:
+                - --health-probe-bind-address=:8081
+                - --metrics-bind-address=127.0.0.1:8080
+                - --leader-elect
+                command:
+                - /manager
+                image: dellemc/dell-csm-operator:v0.1.0
+                imagePullPolicy: Always
+                livenessProbe:
+                  httpGet:
+                    path: /healthz
+                    port: 8081
+                  initialDelaySeconds: 15
+                  periodSeconds: 20
+                name: manager
+                readinessProbe:
+                  httpGet:
+                    path: /readyz
+                    port: 8081
+                  initialDelaySeconds: 5
+                  periodSeconds: 10
+                resources:
+                  limits:
+                    cpu: 200m
+                    memory: 100Mi
+                  requests:
+                    cpu: 100m
+                    memory: 20Mi
+                securityContext:
+                  allowPrivilegeEscalation: false
+              securityContext:
+                runAsNonRoot: true
+              serviceAccountName: dell-csm-operator-controller-manager
+              terminationGracePeriodSeconds: 10
+    strategy: deployment
+  installModes:
+  - supported: true
+    type: OwnNamespace
+  - supported: true
+    type: SingleNamespace
+  - supported: false
+    type: MultiNamespace
+  - supported: true
+    type: AllNamespaces
+  keywords:
+  - Dell CSM Operator
+  - Dell CSI Driver
+  - Dell CSM Modules
+  - Powerscale
+  - Authorization
+  links:
+  - name: Documentation
+    url: https://dell.github.io/csm-docs/docs/deployment/csmoperator/
+  maintainers:
+  - email: dell.csm.operator@dell.com
+    name: Dell CSM Operator
+  maturity: alpha
+  minKubeVersion: 1.21.0
```

Comment:
@rensyct `minKubeVersion: 1.21.0` should mean releasing to `OCP v4.8` and higher. Please add `annotations.com.redhat.openshift.versions: "v4.8"` to `annotations.yaml`

Reply:
Hi @J0zi, support matrix for dell-csm-operator is OCP 4.8 and OCP 4.9. Added `com.redhat.openshift.versions: v4.8  ` to `annotations.yaml`

Reply:
Hi @J0zi 
We use the `replaces` field in CSV. Will the upgrade from `CSM 0.1.0 to CSM 0.2.0 `work as expected on` OCP 4.10` with `replaces` field in CSV if we change it to `4.8-4.9`

Reply:
@rensyct  If I understand it correctly, CSM works also on `v4.10`, so we can merge it with `annotations.com.redhat.openshift.versions: v4.8`, which will provide your operator with all indexes `4.8+`. Please confirm, that I can merge it as it is.

Reply:
Please merge.. thank you @J0zi

Reply:
@rensyct your `olm.maxOpenShiftVersion` is `4.9`, so the label should be aligned with this value. I suggest `annotations.com.redhat.openshift.versions: 4.8-4.9` which will not release to `v4.10` index. 

Or you should change `  olm.properties: '[{"type": "olm.maxOpenShiftVersion", "value": "4.9"}]'` if you want release in `v4.10` index

Reply:
let me summarize:
```
minKubeVersion: 1.21.0
annotations.com.redhat.openshift.versions: 4.8-4.9
'[{"type": "olm.maxOpenShiftVersion", "value": "4.9"}]'
```
this is all aligned ^ and means 4.8-4.9

this is not compatible:
```
minKubeVersion: 1.21.0
annotations.com.redhat.openshift.versions: 4.8
'[{"type": "olm.maxOpenShiftVersion", "value": "4.9"}]'
```

if you are not using deprecated API, you can omit the line containing [{"type": "olm.maxOpenShiftVersion", "value": "4.9"}]


so this is also compatible and means 4.8+:
```
minKubeVersion: 1.21.0
annotations.com.redhat.openshift.versions: 4.8
```

Reply:
For this we dont have 4.10 in the support matrix. so it is right that it should not release to 4.10 index. But next release can we use "replaces" keyword in CSV. Would the operator work as expected on a OCP .10

Reply:
We are not using deprecated APIs. i can omit the below line
[{"type": "olm.maxOpenShiftVersion", "value": "4.9"}]